Python/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 install(
   FILES getkw.py
-  DESTINATION ${CMAKE_INSTALL_LIBDIR}/python2.7/site-packages
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/python${_PYTHON_VERSION}/site-packages
```

Comment:
Where does `_PYTHON_VERSION` come from?

Reply:
It's defined in `cmake/downloaded/autocmake_python_libs.cmake`

Reply:
I think it's better to use `${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}` variables with an underscore are not supposed to be used outside of the file where they are defined.